production/helm/loki/values.yaml

```diff
@@ -14,8 +14,13 @@ affinity: {}
 ## Deployment annotations
 annotations: {}
 
+# enable tracing for debug, need install jaeger and specify jaeger_agent_host
+trace_enabled: false
```

Comment:
Use camel case for variables as is the standard for Helm and Kubernetes. So `traceEnabled` and `jaegerAgentHost`. Variables under config are the exception since they originate from code.